src/Components/AddNewProject/AddNewProject.tsx

```diff
@@ -0,0 +1,111 @@
+
+
+  const navigate = useNavigate();
+  const {baseUrl , requstHeaders} : any = useContext(AuthContext);
+  const [isLoding , setIsLoding] = useState(false);
+
+
+  const {
+      register,
+      handleSubmit,
+      formState : { errors },
+  } = useForm();
+
+  // ****************** to add new project **********************
+  const onSubmit = (data)=> {
+      // console.log(data);
+      setIsLoding(true);
+
+      axios.post(`${baseUrl}/Project` , data ,
+      {
+          headers : requstHeaders,
+      })
+      .then((response)=> {
+          console.log(response);
+          navigate("/dashboard/projects")
+          toast.success( response?.data?.message || "Added Successfully")
+      })
+      .catch((error)=> {
+          console.log(error?.response?.data?.message);
+          toast.error(error?.response?.data?.message || "invalid Data")
+      })
+      .finally(()=> {
+        setIsLoding(false);
+      })
+  }
+
+  // *************** if pressing Cancel ****************
+  const backToProjects = ()=> {
+      navigate("/dashboard/projects");
+  }
+
+
+  return (
+    <>
+      <div className="pb-5">
+        {/* **************** to content header ****************** */}
+        <div className='bg-white header  px-4 py-3 '>
+            <Link className="navigate" to = "/dashboard/projects"> <i className="fa-solid fa-chevron-left"></i> View All Projects </Link>
+            <h3> Add a New Project </h3>
+        </div>
+
+        {/* **************** to view form ****************** */}
+        <form onSubmit={handleSubmit(onSubmit)} className="w-75 shadow bg-white p-4 m-auto mt-4 rounded-3">
+
+          {/* ************** for name input *************** */}
+            <div className='form-group mt-4'>
+              <label>Title</label>
+              <input className='form-control rounded-3 py-2 mt-2'
+                      placeholder= 'Name' 
+                      type="text" 
+                      {...register("title" , {
+                      required: true,
+                  })}
+              />
+
+              {errors.title && errors.title.type === "required" && (
+                <span className='text-danger mt-4'>title is required</span>
+              )}
+
+            </div>
+
+            {/* ************** for description input ************ */}
+              <div className='form-group mt-5 '>
+                <label>Description</label>
+                <input className='form-control rounded-3 pb-5 mt-2'
+                      placeholder= 'Description' 
+                      type="text" 
+                      {...register("description" , {
+                      required: true,
+                  })}
+                />
+
+                {errors.description && errors.description.type === "required" && (
+                    <span className='text-danger mt-4'>description is required</span>
+                )}
+              </div>
+
+              <div className="d-flex justify-content-between mt-5">
```

Comment:
the buttons here don't have a  background color hover color, try choosing a light colors of the current one